src/renderer/renderer_elements.ts

```diff
@@ -860,6 +862,152 @@ export class ReturnBlock extends BasicBlock {
 
 }
 
+export class ConditionalRegion extends ControlFlowRegion {
+    
+    public static get CONDITION_SPACING(): number {
+        return 3 * SDFV.LINEHEIGHT;
```

Comment:
I think this can be adjusted downwards a little bit as it seems quite 'spacious' in the screenshots when the condition is shown.